views_api.py

```diff
@@ -1,29 +1,22 @@
 from http import HTTPStatus
 import asyncio
-import ssl
 import json
-from typing import List
-from fastapi import Request, WebSocket
+from typing import List, Union
+from fastapi import WebSocket, WebSocketDisconnect
 from fastapi.param_functions import Query
 from fastapi.params import Depends
-from fastapi.responses import JSONResponse
-
-from starlette.exceptions import HTTPException
-from sse_starlette.sse import EventSourceResponse
 from loguru import logger
 
 from . import nostrclient_ext
 
-from .tasks import client, received_event_queue
+from .tasks import client, received_event_queue, received_subscription_events
```

Comment:
hmm, is it ok for `views_api` to import from `tasks`?

Reply:
one possibility here: https://github.com/lnbits/nostrmarket/blob/main/__init__.py#L43-L54